README.md

```diff
@@ -1,52 +1,84 @@
+    * [yaml](https://github.com/gslab-econ/gslab_stata) a Stata ado file for parsing yaml files. FIX THIS URL. 
+
+## Getting started
+
+1. Open a shell, clone the repository, and navigate to its root.
+    ```
+    git clone https://github.com/gslab-econ/template.git
+    cd template
+    ```
+2. You're ready to go. We'll prompt you to enter any information necessary and store it in `user-config.yaml` as your scripts run. 
+    * To build everything that has been modified or with dependencies in the repository that have been modified.
+        ```
+        scons
+        ```
+    * To build everything in a single directory of targets that has been modified and all of their dependencies that have been modified.
+        ```
+        scons build/path/to/directory
+        ```
+    * To build a single target that has been modified and all of its dependencies that have been modified.
+        ```
+        scons build/path/to/file.ext
+        ```
+
+## Copying the template
+
+If you want to create a repository with the same structure as this template you can fork it. If you want a repository without any of our git history, follow these instructions. 
+* Create an empty repository in GitHub and clone it. 
+* Copy the contents of this template into the empty repository. Make sure to exclude the `.git` folder, but include the `.gitattributes` and `.gitignore` files. 
+* Create a `user-config.yaml` file and run `scons`. 
+* Commit the changes and push to the new repository.
+
+## FAQ 
 
+##### What is `user-config.yaml`?
+
+Each user is allowed to have different local specifications: We don't put any restrictions on where you keep large files, what you call your executables, or how you manage shared directories. We do need to find these things, and that's what `user-config.yaml` is for. Each user maintains an unversioned [yaml file](http://yaml.org/) with these sorts of specifications. Each script uses its associated yaml-parsing module to read these specifications each time the script is run. 
+
+##### What do I put in `user-config.yaml`?
+
+There's no "default" for `user-config.yaml` because it depends on system specifications and user preferences. Two things we do recommend keeping in `user-config.yaml` are the name of your Stata executable—which we'll prompt you to enter if necessary—and the location of a [SCons cache directory](http://scons.org/doc/2.0.1/HTML/scons-user/c4213.html)—if you have one. An example where Example_User is running a factory-fresh StataMP and has local access to a directory named cache/template on Dropbox would be 
+```YAML
+stata_exec: statamp # StataMP-64.exe on windows
+cache: /Users/Example_User/Dropbox/cache/template
 ```
 
-#### License
+##### Can I only use Stata for data analysis?
+
```

Comment:
I'd suggest phrasing the question in a positive way, like "can we use other softwares" instead